check.js

```diff
@@ -128,14 +128,12 @@ if (config.updatemessage === false || config.updatemessage == "n") {
                 }
                 fs.writeFileSync(path.join(config_dir, "local.json"), DEFAULT_CONFIG);
                 process.exit(0);
-                break;
```

Comment:
???? doesn't thank  make  all command will be executed?

Reply:
OML 

Im so sorry i didnt think the pr would update. 

This code past the first 3 commits is all heavy wip.

I was working on combining the discord commands and the console commands and didnt realize that it updated the pr. 

Dude, im terribly sorry. 

All the current jank in the code wasnt supposed to be here lmao

Reply:
I added deploy-commands.js to load the local guild commands because currently they are not global